layout/_layout.swig

```diff
@@ -10,6 +10,7 @@
   {% include '_partials/head.swig' %}
   <title>{% block title %}{% endblock %}</title>
   {% include '_third-party/analytics/index.swig' %}
```

Comment:
I think `firestore` count need to add into `_third-party/analytics/index.swig` (where `leancloud` already exists) instead of global layout.

Reply:
fixed in e52c4fd15adb8996d3d37fca005ec9162b9065ac

Reply:
So fast fix! :+1: 

But i'm sorry, it was mistake about move to `_third-party/analytics/index.swig`.
Please, add `firestore` in line 84, after `_third-party/analytics/lean-analytics.swig`:
https://github.com/iissnan/hexo-theme-next/blob/89883980c2e6eb7e3600921a12cc879f7426383c/layout/_layout.swig#L83
```diff
+{% include '_third-party/analytics/firestore.swig' %}
```
I think it will right.

Reply:
ok
0935f11e1427179790234d55baa2a8cd4641ac9a